### ISSUE TITLE
Drag & drop edge scroll speed

### ISSUE THREAD

**Issue by ChromaticAberration** [h6ah4i/android-advancedrecyclerview#85]
When using the drag and drop functionality, the auto edge scroll can be a bit too fast for small lists, which makes it tricky to place items correctly.

I believe the values that determine the speed are the private float constants inside the RecyclerViewDragDropManager class.

Can we have setter methods that can tweak these values, or is there a method somewhere else that can change the speed?

**Comment by h6ah4i** [comment]
Will be included in v0.8.0 release.

**Comment by ChromaticAberration** [comment]
Thank you :)

**Comment by h6ah4i** [comment]
Hi. I have just published v0.8.0 and it has introduced these new methods.

- [void RecyclerViewDragDropManager.setDragEdgeScrollSpeed(float speed)](https://github.com/h6ah4i/android-advancedrecyclerview/blob/023f6c0cbcecc9f48f38622a7e8937b10b3d858c/library/src/main/java/com/h6ah4i/android/widget/advrecyclerview/draggable/RecyclerViewDragDropManager.java#L438)

```java
    /**
     * Sets drag edge scroll speed.
     *
     * @param speed The coefficient value of drag edge scrolling speed. (valid range: 0.0f .. 2.0)
     */
    public void setDragEdgeScrollSpeed(float speed) {
        mDragEdgeScrollSpeed = Math.min(Math.max(speed, 0.0f), 2.0f);
    }
```

- [float RecyclerViewDragDropManager.getDragEdgeScrollSpeed()](https://github.com/h6ah4i/android-advancedrecyclerview/blob/023f6c0cbcecc9f48f38622a7e8937b10b3d858c/library/src/main/java/com/h6ah4i/android/widget/advrecyclerview/draggable/RecyclerViewDragDropManager.java#L447)

```java

    /**
     * Gets drag edge scroll speed.
     *
     * @return The coefficient value of drag edges scrolling speed.
     */
    public float getDragEdgeScrollSpeed() {
        return mDragEdgeScrollSpeed;
    }
```
